gui/components/emoji_popup.py

```diff
@@ -6,9 +6,9 @@
 from .base_popup import BasePopup
 
 
-class EmojiPopup(BasePopup):
+class EmojiPopup(QObject):
     def __init__(self):
-        super(EmojiPopup, self).__init__()
+        super(EmojiPopup, self).__init__('mainWallet_AddEditAccountPopup_AccountEmojiSearchBox')
```

Comment:
It can confuse when someone decides to get the object of EmojiPopup, but got SerachBox.
It is better to inherit BasePoup in the EmojiPopup class and create the wait_unit_appears and wait_untill_hidden  methods, where you will wait for SerachBox

Reply:
@StateOf-Vlado this is exactly the reason why i changed that this way. Wait until hidden was not working in that case which leaded to test running 15 minutes

![image](https://github.com/status-im/desktop-qa-automation/assets/82375995/732633dc-25f9-4019-b921-acc33e211990)